src/components/modals/UserModal.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import ModalWithForm from "./ModalWithForm";
+import { UserForm } from "../form";
+import logodark from "../../images/logo-darkbackground.svg";
+export function UserModal({
+  isOpen,
+  onClose,
+  name,
+  title,
+  signUp,
+  submitText,
+  text,
+  linkText,
+  onLinkClick,
+  onSubmitClick,
+  onCancelClick,
+}) {
+  return (
+    <>
```

Comment:
`<>` empty tags are not needed if you have a common tag that wraps the whole markup of the component